### PR TITLE
dokku-client: new port

### PR DESCRIPTION
**Pull request by bashu**
#### Description

Command-line client for the Dokku PaaS

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.6.1 21G217 x86_64
Xcode 14.1 14B47b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by bashu** [comment]
I skipped: `port test ...` cuz there are no tests for this port

File: sysutils/dokku/Portfile [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+github.setup            dokku dokku 0.29.0 v
+github.tarball_from     archive
+
+categories              sysutils
+
+maintainers             {@bashu gmail.com:bashu.was.here} openmaintainer
+
+description             Command-line client for the Dokku PaaS
+
+long_description        ${description}
```
Comment (herbygillot) [inline sysutils/dokku/Portfile:15]: Reusing the `${description}` variable like this requires the use of Tcl's expansion operator (`{*}`), so this should be written like so:

```suggestion
long_description        {*}${description}
```

Comment (bashu) [reply sysutils/dokku/Portfile]: fixed

**Comment by herbygillot** [comment]
I see `dokku` has Go components - is the single shell script really all that needs to be installed?

**Comment by bashu** [comment]
> I see dokku has Go components - is the single shell script really all that needs to be installed?

Yes, we only use one standalone script from contrib subdirectory. On second thought should I rename the package to something like, dokku-cli ? What do you think ?

**Comment by herbygillot** [comment]
Yeah, perhaps it might make more sense to name this port `dokku-client`

**Comment by bashu** [comment]
> Yeah, perhaps it might make more sense to name this port dokku-client

Done

File: sysutils/dokku-client/Portfile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                        size    2137220
+
+depends_run             path:bin/bash:bash
+
+use_configure           no
+
+build                   {}
+
+destroot {
+    xinstall -m 755 ${worksrcpath}/contrib/dokku_client.sh ${destroot}${prefix}/bin/dokku
```
Comment (herbygillot) [inline sysutils/dokku-client/Portfile:34]: Perhaps the installed binary should also be renamed to `dokku_client`?

Comment (herbygillot) [reply sysutils/dokku-client/Portfile:34]: Also make the permission `0755`

Comment (bashu) [reply sysutils/dokku-client/Portfile]: > Also make the permission 0755

Fixed

**Review by herbygillot — APPROVED** [review]

**Comment by herbygillot** [comment]
Merged. Thanks both for this submission, and for working through these changes.

**Pull request merged**